www/install.rst

```diff
@@ -29,34 +29,28 @@ Installing via pip
 ------------------
 
 Mac and Linux users can install pre-built binary packages for the SciPy stack
-using `pip <https://pip.pypa.io/en/stable>`_.  pip is the installation tool
-recommended by the `Python Packaging Authority <https://www.pypa.io>`_. Pip
-can install pre-built binary packages in the `wheel
-<http://wheel.readthedocs.io>`_ package format.
+using `pip <https://pip.pypa.io/en/stable>`_.  Pip can install pre-built binary
+packages in the `wheel <http://wheel.readthedocs.io>`_ package format.
+
+*Note that you need to have Python and pip already installed on your system.*
 
 *pip does not work well for Windows* because the standard pip package index
-site, `pypi <https://pypi.python.org/pypi>`_, does not yet have Windows wheels
+site, `PyPI <https://pypi.python.org/pypi>`_, does not yet have Windows wheels
 for some packages, such as SciPy.
 
-To install via pip on Mac or Linux, first upgrade pip to the latest version.
-For Python 3::
-
-    python3 -m pip install --upgrade pip
+To install via pip on Mac or Linux, first upgrade pip to the latest version::
 
-Replace ``python3`` with ``python2`` for Python 2 pip upgrade.
+    python -m pip install --upgrade pip
 
 Then install the SciPy stack packages with pip.  We recommend a *user*
 install, using the ``--user`` flag to pip.  This installs packages for your
 local user, and does not need extra permissions to write to the system
 directories::
```

Comment:
I think it'd be worth stressing explicitly that `sudo pip` is *not* recommended. The internet is unfortunately full of recommendations to do just that.